cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/InMemoryDatasetFramework.java

```diff
@@ -181,7 +181,6 @@ public synchronized void addInstance(String datasetType, Id.DatasetInstance data
     }
     DatasetDefinition def = registry.get(datasetType);
     DatasetSpecification spec = def.configure(datasetInstanceId.getId(), props);
-    LOG.info("Dataset Specification name {} and properties {}", spec.getName(), spec.getProperties());
```

Comment:
why remove? I thought you added it for a reason. I'd though say "Creating dataset with spec name {} and properties {}..."